modules/auth/saas/settings/subscriptions.tsx

```diff
@@ -6,12 +6,103 @@ import Spinner from "@/resources/containers/spinner";
 import Service from "@/core/module/service.factory";
 import { useRouter } from "next/router";
 import moment from "moment";
+import Table from "@/resources/table";
```

Comment:
I just realize, this subscriptions is not in the `saas` settings, need to be in the `subscriptions module` 
I moved the lang of the previous PR already, lets make sure the lang file is flat (for the IA be able to manage it easily)

Reply:
I think you were able to make nested string work?